rpi_csdt_community/templates/rpi_csdt_community/classrooms_guide.html

```diff
@@ -0,0 +1,41 @@
+            </p>
+            <p>
+                To create a classroom, first log in to your CSDT account. Any account can make a classroom. Click classrooms at the top and then find "Create Your Own" at the upper right, just below "Log Out". Here you can name your classroom and provide a short description. The name is important because, other than your username, it is the only identifier with which your students can find your classroom. If you would like your classroom to be private (not viewable by visitors), please click "Private?" After you click "create class" you will be taken to your classroom page. Notice the number of your classroom in the URL. This number is your classroom number. You can tell your students when they register with CSDT your classroom number, which is a registration question, and they can automatically be added to your classroom when registering. Otherwise, your students will have to find your classroom in the classroom list and click "join".  Newest classrooms are at the bottom of the list.
+            </p>
+            <p>
+                When a student requests to join your class, a request notification will be sent to your administration panel. To go to your administration panel, go to the classrooms page. Your own classrooms will be at the top of the list. Click "Admin" to the right of your classroom title. Under the member-request panel, all people who have asked to join your classroom await approval. To group approve, click their checkmark above their icon for as many as your want to approve, then under the "action" dropdown choose "Approve" and click "Submit all changes". They will promoted to the "students" panel. As a student, when they save their project, there will be a bar where they can choose whether to save to their classroom. They will always want to use this option, which requires them to highlight the name of their classroom. Teach them how to use this feature to prevent projects being saved to the public pool instead of your classroom pool.
```

Comment:
Awkward phrasing: "To group approve, click their checkmark above their icon for as many as your want to approve," -> "To approve students, click the checkmark above their icon"